creativecloud/scripts/utils.js

```diff
@@ -108,7 +108,7 @@ function getDecorateAreaFn() {
 
   return (area, options) => {
     if (isRootPage()) replaceDotMedia();
-    if (!lcpImgSet) loadLCPImage(area, options);
+    loadLCPImage(area, options);
```

Comment:
can we change it to check if options have fragment link and if (!lcpImgSet) only then call loadLCPImage

Reply:
The same check anyways I am doing in loadLCPImage function at the start itself. But, ill can add them here as well